src/Map.jsx

```diff
@@ -75,7 +75,21 @@ class Map extends Component {
       { force: true },
     );
   }
-
+  getRouteBetweenStops(route) {
+    const stopSids = this.state.selectedStops.map(stop => stop.sid);
+    const routeStops = route.stops;
+    const indexArray = [];
+    for (let i = 0; i < routeStops.length; i += 1) {
+      if (stopSids.includes(routeStops[i].sid)) {
+        indexArray.push(i);
+        if (indexArray.length === 2) {
+          break;
+        }
+      }
+    }
+    indexArray.sort((a, b) => a - b);
+    return route.stops.filter((stop, index) => index >= indexArray[0] && index <= indexArray[1]);
```

Comment:
nit - seeing => and <= seems a bit confusing, could you format this along multiple lines?